src/app/screens/LNURLWithdraw/index.tsx

```diff
@@ -62,15 +62,14 @@ function LNURLWithdraw() {
 
       if (response.data.status.toUpperCase() === "OK") {
         setSuccessMessage("Withdraw request sent successfully.");
+        // ATTENTION: if this LNURL is called through `webln.lnurl` then we immediately return and return the response. This closes the window which means the user will NOT see the above successAction.
```

Comment:
@bumi this seems to do the trick. Do you see any issues with this?  
I could apply this to the other prompts.

Reply:
yep, I guess something like that 👍 
either moving the reply up. or extending the `if (navState.isPrompt) {` around the reply.